Linux.md

```diff
@@ -2,36 +2,80 @@
+- **Built for development.**
+As Linux was originally made for developers by developers, they have spent much time and effort perfecting the tools that they would come to use. It has a powerful [shells](https://en.wikipedia.org/wiki/Unix_shell) that can be used for a variety of both programming and administrative tasks ([Bash][4] is the most popular and default choice for Linux).
+While Linux has a "notepad" equivalent in [gedit](https://en.wikipedia.org/wiki/Gedit), it also provides more powerful and customizable text-based editors such as [Vim][5] and [Emacs](https://en.wikipedia.org/wiki/Emacs) (It is recommended Linux users know at least one of these editors). There are also IDEs available for web-development such as [Atom](https://atom.io/), [Aptana](http://www.aptana.com/)/[Eclipse](https://eclipse.org/), [Sublime](https://www.sublimetext.com/), [KomodoIDE](http://komodoide.com/), to name a few.
+Linux also can utilize software such as [Apache](https://httpd.apache.org/) to setup a local web server for testing, [Git](https://git-scm.com/) for version control, and other tools/languages such as [Node.js](https://nodejs.org/en/)/[Ruby](https://www.ruby-lang.org/en/)/[SaSS](http://sass-lang.com/)/[Heroku](https://www.heroku.com/) all have support on Linux and have command line tools that can be used in lieu of a GUI.
+- **It's Free.**
+  Hard to beat that!
 
-> Linux is simply everywhere.
+[Back to top](#table-of-contents)
+
+## How to get Linux
+
+While most vendors provide one OS that gets incremental updates and new versions over time (e.g. Windows, Mac OSX, Solaris), Linux has hundreds of different versions to choose from created by different vendors. This is in part because Linux is freely-distributed code which can be tailored to specific user's needs. While the [Linux Kernel](https://en.wikipedia.org/wiki/Linux_kernel) typically stays the same (core functionality of the OS handling tasks like network management, disk usage, memory utilization, I/O etc.), the many different versions of Linux differ in parts such as which GUI it uses (i.e. [Gnome](https://en.wikipedia.org/wiki/Gnome), [KDE](https://en.wikipedia.org/wiki/KDE), [XFCE](https://en.wikipedia.org/wiki/Xfce), or none at all), system tools, environments, package managers, and applications provided. This unique collection of software along with the kernel define what we call a [Linux Distribution](https://en.wikipedia.org/wiki/Linux_distribution) (distro for short). 
```

Comment:
would be great if you can link the [Linux kernel repository](https://github.com/torvalds/linux) here. Also link the official webpages for [Gnome](https://www.gnome.org/), [KDE](https://www.kde.org/), and [XFCE](http://www.xfce.org/)